pkg/controller/cluster/resource/resource.go

```diff
@@ -350,10 +350,34 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.Cluster, si
 		},
 	}
 
+	sysctlContainer := sysctlInitContainer(c.Spec.Sysctls)
+	if sysctlContainer != nil {
+		sts.Spec.Template.Spec.InitContainers = append(sts.Spec.Template.Spec.InitContainers, *sysctlContainer)
+	}
 	sts.Spec.Template.Spec.Containers = append(sts.Spec.Template.Spec.Containers, *agentContainer(c))
```

Comment:
Can this go as [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/).

Reply:
I don't follow. They are kubernetes init containers.

Reply:
Sorry my bad missed the line